node_modules/oae-core/signin/signin.html

```diff
@@ -1,81 +1,126 @@
 <!-- CSS -->
 <link rel="stylesheet" type="text/css" href="css/signin.css" />
+<link rel="stylesheet" type="text/css" href="css/select2.css" />
```

Comment:
As mentioned previously, this should really be in the shared folder - otherwise we won't be able to use the lib elsewhere.